docs/encrypted_images.md

```diff
@@ -69,13 +69,36 @@ but randomizing a 16-byte block with a TRNG should make it highly
 improbable that duplicates ever happen.
 
 To distribute this AES-CTR-128 key, new TLVs were defined. The key can be
-encrypted using either RSA-OAEP or AES-KW-128. Also in the future support
-for EICES (using EC) can be added.
+encrypted using either RSA-OAEP, AES-KW-128 or ECIES.
```

Comment:
This use of ECIES is less ambiguous, because we are at least saying we are using it to wrap the key.  I wonder if something like ECIES-KW might be less confusing, though.

Reply:
I updated all references to be either `ECIES_P256` or `EC256`. It does not feel appropriate to add `-KW` because that terminology does not exists in the papers. But the docs are somewhat clear of what we are doing.